configs/platforms/windows-2019-x64.rb

```diff
@@ -5,15 +5,16 @@
   visual_studio_version = '2017'
   visual_studio_sdk_version = 'win8.1'
 
+  choco_installer = 'https://chocolatey.org/install.ps1'
+  choco_version = '1.4.0'
   # We need to ensure we install chocolatey prior to adding any nuget repos. Otherwise, everything will fall over
-  plat.add_build_repository "https://artifactory.delivery.puppetlabs.net/artifactory/generic/buildsources/windows/chocolatey/install-chocolatey.ps1"
+  plat.provision_with %(powershell.exe -NoProfile -ExecutionPolicy Bypass -Command '$env:chocolateyVersion="#{choco_version}";iex ((new-object net.webclient).DownloadString("#{choco_installer}"))')
```

Comment:
I ran into this problem too, but we still use win2012R2 with an old .NET version which can't connect to chocolatey.org using TLS 1.2. So I pushed a new `install-chocolatey-1.4.0.ps1` to artifactory and updated our platform definitions to use that https://github.com/puppetlabs/puppet-runtime/pull/679